playbooks/robusta_playbooks/job_actions.py

```diff
@@ -0,0 +1,92 @@
+from robusta.api import *
+
+
+class JobParams(ActionParams):
+    """
+    :var image: The job image.
+    :var command: The job command as array of strings
+    :var name: Custom name for the job and job container.
+    :var namespace: The created job namespace.
+    :var restart_policy: Job container restart policy
+    :var job_ttl_after_finished: Delete finished job ttl (seconds). If omitted, jobs will not be deleted automatically.
+    :var notify: Add a notification for creating the job.
+
+    :example command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+    """
+    image: str
+    command: List[str]
+    name: str = "robusta-action-job"
```

Comment:
I think that requiring an explicit name from the user is much preferred (removing the default "robusta-action-job"). Will make it searchable in the values file and will make sure that the user understands he creates a job.

Reply:
I rather leave the default name, so the user can have less parameters to configure